### PR TITLE
Alumni page addition

### PR DESCRIPTION
**Pull request by ArkaprabhaChakraborty**
solving issue #11. Still some work is left as alumni helper has to be updated. Ill suggest till then this PR can be made a draft.

### CODE REVIEW

**Comment by aniruddha2000** [comment]
LGTM :-)

File: components/helper/teamHelper.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+},
+{
+    name : "Mr. Subhadip Chandra",
+    image : "https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fsubhadipsir.jpeg?alt=media&token=0ffc1ea1-0979-4107-ac03-054637a2d7f9",
+    linkedin : "https://www.linkedin.com/mwlite/in/subhadip-chandra-17b82592",
+    pos : "Faculty Advisor"
+}]
+
+export const coreLeads = [
+    {
```
Comment (aniruddha2000) [inline components/helper/teamHelper.js:16]: Edit1 : Updated with latest Firebase URL
```
Leads
Arkaprabha - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Farkaprabha_chakraborty_o1OjlTV.webp?alt=media&token=d0a7474c-a383-4688-8df9-ab08d8a7a037
Arko - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Farko_sen_gaqqaYp.jpg?alt=media&token=211c1c5a-8793-4a22-88a5-65ed97c77f5e
Rajarshi - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Frajarshi_chakraborty_ERAxKAj.webp?alt=media&token=7898b4b8-0803-42f2-8f3c-1014c0fefa17
Rohan - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Frohan_rudra_sdpY4Cl.webp?alt=media&token=05b31d03-1c70-4ddf-95fd-07f0a9fd6be1
```

**Review by aniruddha2000 — CHANGES_REQUESTED** [review]
There are the cdn link update the fields with their github url.

File: components/helper/teamHelper.js (hunk 1 above)
Comment (aniruddha2000) [reply components/helper/teamHelper.js:16]: Made you the collaborator of DSC interview form where you can take the github link :)

File: components/helper/teamHelper.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        name : "Akashdeep Bhattacharya",
+        linkedin : "https://www.linkedin.com/in/akashdeep-bhattacharya-8aa417158/",
+        github : "https://github.com/akashdeep007",
+        image : 'https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fakashdeep.jpg?alt=media&token=d5cf4e66-086d-40cb-8d94-f5d9387c16dc',
+        pos : "Android Lead"
+    },
+]
+
+export const coreMembers = [
+    {
```
Comment (aniruddha2000) [inline components/helper/teamHelper.js:51]: With the latest Firebase URL
```
Volunteers - 
Ahana - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fahana_basu_IYhngox.webp?alt=media&token=4672a383-2c94-4956-bade-7abaebaced53
Chandrima - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fchandrima_pal.webp?alt=media&token=2ef18174-90bc-4d35-b01f-53c635f5bfab
Maaitryo - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fmaaitrayo_das_ditNPDZ.webp?alt=media&token=2df0d1ca-9cf8-4f50-ab63-ef2400dd1fc0
Shams - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fmd_shams_firoz.webp?alt=media&token=e6b2555e-1a68-49f0-8d8a-0319636c0500
```

**Review by aniruddha2000 — CHANGES_REQUESTED** [review]
Update the volunteers and same as above for the GitHub

File: components/helper/teamHelper.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+export const lead = [{
+    name : "Aniruddha Basak",
+    image : "https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2FAniruddhaBasak.jpg?alt=media&token=91019834-b7fa-494e-bd57-c940ebd58cf1",
```
Comment (aniruddha2000) [inline components/helper/teamHelper.js]:
```suggestion
    image : "https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Faniruddha_basak_Zb3Nr4P.webp?alt=media&token=e96a1baa-27ca-4f35-808c-c45672cf8701",
```

**Review by aniruddha2000 — CHANGES_REQUESTED** [review]
My photo change :)

File: components/helper/teamHelper.js (hunk 2 above)
Comment (aniruddha2000) [reply components/helper/teamHelper.js:51]:
```
Diganta - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fdiganta_ghosh_Y9EpQ81.jpeg?alt=media&token=1161a4e2-159f-464b-9c95-0f6960b9edc4
Debjita - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fdebjita_mondal.jpg?alt=media&token=6b96793a-7d90-4eb5-a49f-90e5addf89f9
```
Additional 2 volunteers

**Comment by ArkaprabhaChakraborty** [comment]
You can merge it now.

**Comment by aniruddha2000** [comment]
Ok LGTM :)

**Pull request merged**